wolox-books/src/components/BookList.vue

```diff
@@ -1,6 +1,6 @@
 <template lang='pug'>
   .books-container.row
-    .book.column(v-for='book in books' :key='book.id')
+    router-link.book.column(v-for='book in books' :key='book.id' :to='`/books/${book.id}`')
```

Comment:
`:to='{ name: routes.books, params: { ID_PARAM: book.id }}'`
`ID_PARAM` will be replaced depending on the way you defined the path in the router file. If you put something like this `path: '/books/:id'`, then, you should send this in the params: `params: { id: book.id }`